dev_docs/RELEASES.md

```diff
@@ -17,15 +17,15 @@ For patch releases, only the version on the existing major and minor version bra
 
 * Trigger release workflow manually
   * For **patch releases**: run the [`run-patch-release`](https://github.com/elastic/apm-server/actions/workflows/run-patch-release.yml) workflow (In "Use workflow from", select `main` branch. Then in "The version", specify the **upcoming** patch release version - es: on `8.14.2` feature freeze you will use `8.14.2`).
-    This workflow will: create the release branch; update version across codebase; commit and create PR targeting the release branch.
-    Release notes for patch releases **must be manually added** (PR should target `main` branch and backported to the release branch):
+    This workflow will: create the `update-<VERSION>` branch, update version constants across the codebase and create a PR targeting the release branch. 
+    Release notes for patch releases **must be manually added** (PR should target `main` branch and should be backported to the release branch):
```

Comment:
I would add here **must be manually added a day before release**

Reply:
Teams do it on either D-1 or release day. I think it is fine to specify it as D-1 here for clarity.